ui/src/app/job-details/panels/panels.component.css

```diff
@@ -26,10 +26,21 @@ p {
   margin-right: 3rem;
 }
 
-.mat-card {
-  width: 25%;
+.card {
   margin-right: 1rem;
   margin-top: 2rem;
   float: left;
   min-height: 8.7rem;
 }
+
+.card.status {
+  width: 20%;
+}
+
+.card.input-output {
+  width: 30;
+}
+
+.input-output-table {
```

Comment:
Should the table / card contents be scrollable? Is it already?

Reply:
This disables the scroll bar which was weirdly appearing even when there was nothing to scroll. I'm going to ask marty today about whether he thinks it should be scrollable. 

I imagine we should show some default number of rows and either have a button to expand it or make it scrollable.